volatility3/framework/symbols/linux/extensions/__init__.py

```diff
@@ -480,6 +578,15 @@ def get_ptrace_tracee_flags(self) -> Optional[str]:
             else None
         )
 
+    @property
+    def state(self):
```

Comment:
Just spotted this one may also overload.  Probably ok since it's just a pass through, just worried about giving people and example when they try and do worse, what do you think?